src/Components/WebView/Samples/PhotinoPlatform/src/BlazorWindow.cs

```diff
@@ -67,15 +73,138 @@ public class BlazorWindow
     /// </summary>
     public BlazorWindowRootComponents RootComponents { get; }
 
+    private string? _latestControlDivValue;
+
     /// <summary>
     /// Shows the window and waits for it to be closed.
     /// </summary>
-    public void Run()
+    public void Run(bool isTestMode = false)
     {
+        const string NewControlDivValueMessage = "wvt:NewControlDivValue";
+        var isWebViewReady = false;
+
+        if (isTestMode)
```

Comment:
Alright PhotinoPlatform is unchanged except for 2 things:
1. Changes required due to updating the Photino.NET version
2. Exposing PhotinoWindow so that tests can hook into it to do things (listen to events, send messages, etc.)